### PR TITLE
Added chain names

### PR DESCRIPTION
**Pull request by Kayanski**
This PR aims at adding chain names to cw-orch structures

### CODE REVIEW

File: packages/cw-orch-networks/src/chain_info.rs [hunk 1]
```diff
@@ -13,7 +13,11 @@ use cw_orch_core::{
 impl Into<RegistryChainInfo> for ChainInfo<'_> {
     fn into(self) -> RegistryChainInfo {
         RegistryChainInfo {
-            chain_name: self.network_info.id.to_string(),
+            chain_name: match self.kind {
+                ChainKind::Local => self.chain_name.to_string(),
+                ChainKind::Mainnet => self.chain_name.to_string(),
+                ChainKind::Testnet => format!("testnets/{}", self.chain_name),
+            },
```
Comment (Buckram123) [inline packages/cw-orch-networks/src/chain_info.rs:20]:
```suggestion
            chain_name: self.chain_name,
```
I would prefer to just return it as is, IMO we should have as close result as possible to fetching this `chain_name`. User can add path themself based of `ChainKind`, if he needs to. 
image

**Review by Buckram123 — CHANGES_REQUESTED** [review]
Thanks for adding that, now I can easily resolve testnet explorers for CLI :)
I don't think we should resolve path of testnets for user

File: packages/cw-orch-networks/src/chain_info.rs (hunk 1 above)
Comment (CyberHoward) [reply packages/cw-orch-networks/src/chain_info.rs:20]: Agree with this, we shouldn't deflect from chain registry with this.

**Review by CyberHoward — CHANGES_REQUESTED** [review]
I don't think we should do this change. See linear.

**Comment by Kayanski** [comment]
Closed in favor of https://github.com/AbstractSDK/cw-orchestrator/pull/375
See ORC-119

**Pull request closed without merging**